### PR TITLE
Update the exporter IDE list

### PR DESCRIPTION
**Pull request by TacoGrandeTX**
Added: VSCode, Code::Blocks, CrossCore, NetBeans, Qt Creator
Removed: Atmel Studio, Cocox, DS-5, Kinetis Design Studio
Renamed: LPCXpresso --> MCUXpresso, Emblocks --> Embitz

Alphabetized the less popular IDEs.

### CODE REVIEW

**Comment by TacoGrandeTX** [comment]
@theotherjimmy I think we need to strike Simplicity Studio from the list too?  A customer just inquired about that but I didn't notice it was missing from the CLI output.  I see the conversation here:  https://github.com/ARMmbed/mbed-cli/issues/389

**Comment by theotherjimmy** [comment]
@TacoGrandeTX That's correct. We don't have a simplicity exporter.

File: docs/tools/toolchains/export_to_third_party.md [hunk 1]
```diff
@@ -4,24 +4,28 @@ If you'd like to develop on Arm Mbed OS with a third party tool, or migrate to o
...
-- LPCXpresso.
-- Kinetis Design Studio.
+- Code::Blocks.
+- Cross Core Embedded Studio.
+- e2studio.
+- Embitz.
+- MCUXpresso.
+- NetBeans.
+- Qt Creator.
 - Simplicity Studio.
```
Comment (theotherjimmy) [inline docs/tools/toolchains/export_to_third_party.md]: No longer supported.

**Comment by TacoGrandeTX** [comment]
@theotherjimmy Removed Simplicity Studio and two dead sections at the end (on Atmel Studio and KDS). Are we good?

**Review by theotherjimmy — APPROVED** [review]

**Review by AnotherButler — APPROVED** [review]
Thanks for the PR 👍

**Pull request merged**